src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -281,22 +290,17 @@ private function createQueryBuilder(): QueryBuilder
             ->from($this->configuration['table_name'], 'm');
     }
 
-    private function executeQuery(string $sql, array $parameters = [])
+    private function executeQuery(string $sql, array $parameters = [], ?array $types = [])
```

Comment:
why does $types need to be nullable?

Reply:
Without that, few tests does not passes.

```sh
1) Symfony\Component\Messenger\Tests\Transport\Doctrine\ConnectionTest::testGetAMessageWillChangeItsStatus
TypeError: Argument 3 passed to Symfony\Component\Messenger\Transport\Doctrine\Connection::executeQuery() must be of the type array, null given, called in /home/roukmoute/workspace/symfony/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php on line 150
```

Would you prefer I change theses tests to pass without this nullable?

Reply:
The mocks probably need to be fixed so it's does not return a null which violates the contract.